tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,392 @@
+    Arsenic().get(os.environ['SERVER_NAME'] + '/literature/new')
+
+
+@when('I go to the holding panel list page')
+def go_to_holding_pen_literature_list():
+    Arsenic().get(os.environ['SERVER_NAME'] + '/holdingpen/list/?page=1&size=10&status=HALTED&workflow_name=HEP')
+
+
+@when(parsers.cfparse('I click on the button with id {button_id}'))
+def click_generic_button(button_id):
+    WebDriverWait(Arsenic(), 2).until(
+        EC.visibility_of_element_located((By.ID, button_id))
+    ).click()
+
+
+@when(parsers.cfparse('I click on the button link with text {text_value}'))
+def click_generic_link_button(text_value):
+    Arsenic().find_element_by_link_text(text_value).click()
+
+
+@when('I click on the submit button')
+def click_submit_button():
+    WebDriverWait(Arsenic(), 2).until(
+        EC.visibility_of_element_located((By.XPATH, '//div[@id="webdeposit_form_accordion"]/div[4]/span/button'))
+    ).click()
+
+
+@when('I click on the accept button')
+def click_accept_button():
+    WebDriverWait(Arsenic(), 2).until(
+        EC.visibility_of_element_located((By.XPATH, '//button[@class="btn btn-warning"]'))
+    ).click()
+
+
+@when('I click on the button Skip, and fill the form manually')
+def click_skip_and_fill_manually():
+    Arsenic().hide_title_bar()
+    WebDriverWait(Arsenic(), 5).until(
+        TryClick((By.ID, 'skipImportData'))
+    ).click()
+    WebDriverWait(Arsenic(), 5).until(
+        EC.text_to_be_present_in_element(
+            (By.ID, 'form_container'),
+            'Type of Document',
+        )
+    )
+    Arsenic().find_element_by_link_text('Conference Information').click()
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[7]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[8]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[9]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().show_title_bar()
+
+
+@when(parsers.cfparse('I select the value {input_select} in the select box with id {field_id}'))
+def select_generic_value(input_select, field_id):
+    Select(Arsenic().find_element_by_id(field_id)).select_by_value(input_select)
+
+
+@when(parsers.cfparse('I select the value {input_select} in the subject dropdown list'))
+def select_generic_value_dropdown(input_select):
+    Arsenic().find_element_by_xpath('(//button[@type="button"])[8]').click()
```

Comment:
Since this file is all about the literature suggestion form you can drop literature from the name. Something like:

`select_value_in_subject_multiselect_widget`

can work.